packages/truffle-contract/test/linking.js

```diff
@@ -77,26 +107,41 @@ describe("Library linking with contract objects", function() {
   var accounts;
   var web3;
   var network_id;
-  var provider = TestRPC.provider({logger: log});
+  var provider = TestRPC.provider({ logger: log });
   web3 = new Web3();
   web3.setProvider(provider);
 
   before(function() {
-    return web3.eth.net.getId().then(function(id){
+    return web3.eth.net.getId().then(function(id) {
       network_id = id;
     });
   });
 
   before(function() {
     this.timeout(10000);
 
+    const exampleLibraryPath = path.join(
```

Comment:
for some reason these paths weren't resolving correctly while debugging in vs code - this fixes that